### PR TITLE
nodejs: add version 8.11.4

### PR DESCRIPTION
**Pull request by darmac**

### CODE REVIEW

File: var/spack/repos/builtin/packages/node-js/package.py [hunk 1]
```diff
@@ -105,7 +106,12 @@ def configure_args(self):
         return args
 
     def configure(self, spec, prefix):
-        python('configure.py', *self.configure_args())
+        # Adapted to version 8.11.4, none configure.py script
+        if '@8.11.4' in self.spec:
+            sh = which('bash')
+            sh('./configure', *self.configure_args())
+        else:
+            python('configure.py', *self.configure_args())
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/node-js/package.py:110]: It looks like it's even more complicated than this. If you look at version 6, it only has a `configure` script, but it's a Python script. At some point they switch to a shell script named `configure`. Later, they add a `configure.py` script. Can you figure out which versions make these switches and then support all versions?

Comment (darmac) [reply var/spack/repos/builtin/packages/node-js/package.py:110]: Fixed in https://github.com/spack/spack/pull/17824/commits/f541cd16fc77646f8d9906b8b6790176c8e4f7bd

Comment (adamjstewart) [reply var/spack/repos/builtin/packages/node-js/package.py:110]: I think you need a third case for `sh('configure', ...)`

Comment (darmac) [reply var/spack/repos/builtin/packages/node-js/package.py:110]: looks all of the `configure` file is a `python` script?
So I think python('configure') is better?

Comment (adamjstewart) [reply var/spack/repos/builtin/packages/node-js/package.py:110]: Ah, I see. The shebang is `/bin/sh`, but it re-executes the script in Python like our `bin/spack` file does.

**Review by adamjstewart — APPROVED** [review]

**Pull request merged**